src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Description, GetArea {
```

Comment:
```suggestion
public class Circle extends Figure {
```
we have already implemented these interfaces in class Figure, it is enough